lib/runner/process-listener.js

```diff
@@ -49,7 +49,9 @@ module.exports = class {
   }
 
   onExit(code) {
-    this.process.exitCode = code || this.exitCode;
+    if (this.process.exitCode !== 1) {
```

Comment:
how about this instead?
```js
if (this.process.exitCode !== 0) {
   this.setExitCode(this.process.exitCode);
}

this.process.exitCode = code || this.exitCode;
```

Reply:
> how about this instead?
> 
> ```js
> if (this.process.exitCode !== 0) {
>    this.setExitCode(this.process.exitCode);
> }
> 
> this.process.exitCode = code || this.exitCode;
> ```

Yes, based on the exit code value can be 1-255, that will be better.

Reply:
Ok, then if you can make the change, I will merge this PR. Thanks.

Reply:
@beatfactor Thanks. It's updated!